api/src/main/java/net/md_5/bungee/api/event/ServerSwitchEvent.java

```diff
@@ -5,6 +5,7 @@
 import lombok.ToString;
 import net.md_5.bungee.api.connection.ProxiedPlayer;
 import net.md_5.bungee.api.plugin.Event;
+import net.md_5.bungee.api.connection.Server;
```

Comment:
BungeeCord requires imports to be sorted.

Reply:
I fixed that, thank you